warp10/src/main/java/io/warp10/standalone/StandaloneMemoryStore.java

```diff
@@ -748,10 +749,14 @@ private void load(String path) throws IOException {
         encoder.setCount(wrapper.getCount());
         
         bytes += value.getLength() + key.getLength();
-        encoder.safeSetMetadata(wrapper.getMetadata());
+        if (wrapper.isSetMetadata()) {
+          encoder.safeSetMetadata(wrapper.getMetadata());
```

Comment:
Same question as previously: is it safe?

Reply:
Same answer ;-)